components/post_view/post_info/post_info.tsx

```diff
@@ -252,13 +263,12 @@ export default class PostInfo extends React.PureComponent {
         );
     };
 
-    handleShortcutReactToLastPost = (isLastPost) => {
+    handleShortcutReactToLastPost = (isLastPost: boolean) => {
         if (isLastPost) {
-            const {post, isReadOnly, enableEmojiPicker, isMobile,
-                actions: {emitShortcutReactToLastPostFrom}} = this.props;
+            const {post, isReadOnly, enableEmojiPicker, isMobile, actions} = this.props;
 
             // Setting the last message emoji action to empty to clean up the redux state
-            emitShortcutReactToLastPostFrom(Locations.NO_WHERE);
+            actions?.emitShortcutReactToLastPostFrom?.(Locations.NO_WHERE);
```

Comment:
`?` isn't needed here anymore.

Reply:
thanks @devinbinnie - removed 2 such instances in the commit I just pushed.